### PR TITLE
#381 Apply Conan conventions

### PR DESCRIPTION
**Pull request by uilianries**
/cc @danimtb

### CODE REVIEW

File: conanfile.py [hunk 1]
```diff
@@ -13,67 +13,53 @@ class ProtobufConan(ConanFile):
...
+        os.rename(extracted_dir, self._source_subfolder)
 
-        #Rename to "source_subfolder" is a convention to simplify later steps
-        os.rename(extracted_dir, self.source_subfolder)
-
-    def configure_cmake(self):
+    def _configure_cmake(self):
         cmake = CMake(self)
-        cmake.definitions["CMAKE_INSTALL_LIBDIR"] = "lib"
         cmake.definitions["protobuf_BUILD_TESTS"] = self.options.build_tests
```
Comment (danimtb) [inline conanfile.py]: This option is set always to False in 3.6.1 and the recipe does not have such ``build_tests`` option. Option and usage are ok, however the binaries generated are the same when the option is true or false but the package ID will change. Options like this one should not change package ID:

```
def package_id(self):
    del self.info.options.build_tests
```

Comment (uilianries) [reply conanfile.py]: Maybe we could exclude that option and force to False. WDYT?

Comment (danimtb) [reply conanfile.py]: Yes, that is what I am trying to say: Set it always to False or remove the option from package ID

But this is not a big issue if you want to keep it as is, just wanted to highlight it

Comment (uilianries) [reply conanfile.py]: understood. I'll remove it

File: conanfile.py [hunk 2]
```diff
@@ -13,67 +13,53 @@ class ProtobufConan(ConanFile):
...
         cmake.definitions["protobuf_BUILD_PROTOC_BINARIES"] = self.options.build_binaries
```
Comment (uilianries) [inline conanfile.py]: Should we keep build binaries? Now we have protoc_installer ... IMO it could be removed.

Comment (uilianries) [reply conanfile.py]: replaced by proto_installer as requirement

File: conanfile.py (hunk 1 above)
Comment (uilianries) [reply conanfile.py]: Done!

**Pull request merged**